lib/fluent/plugin/in_rds_mysql_log.rb

```diff
@@ -1,7 +1,18 @@
+require 'fluent:input'
```

Comment:
Sorry for nitpick review comment...
Please don't consider, I don't want to merge your PR.
I appreciate your contribution 😄 

Honestly, I don't have enough time to maintain this plugin, and I'm mostly satisfied with the feature.

> I pushed the changes up to ruby gems and plan to make quite a few more in the plugin itself.

If you maintain this plugin, it's really great!!
I'll switch to use your plugin.